README.md

```diff
@@ -139,52 +134,112 @@ Previous simulation platforms that have operated on similar datasets typically p
 
 To run the above benchmarks on your machine, see instructions in the [examples](#examples) section.
 
-## Quick installation
+## Installation
+
+### Docker Image
+
+We provide a pre-built docker container for habitat-api and habitat-sim, refer to [habitat-docker-setup](https://github.com/facebookresearch/habitat-api#docker-setup).
+
+### From Source
+
+We highly recommend installing an [Anaconda](https://www.anaconda.com/distribution/#download-section) environment (note: python>=3.6 is required). Once you have Anaconda installed, here are the instructions.
+
+
+1. Clone this github repository. 
+   ```bash
+   git clone https://github.com/facebookresearch/habitat-sim.git
+   cd habitat-sim
+   ```
+
+1. Install Dependencies 
+  
+    Common
+   ```bash
+   # We require python>=3.6 and cmake>=3.10
+   conda create -n habitat python=3.6 cmake=3.14.0
+   conda activate habitat
+   pip install -r requirements.txt
+   ```
+
+    Linux (Tested with Ubuntu 18.04 with gcc 7.4.0)
+   ```bash
```

Comment:
Can we add a comment like `These are fairly ubiquitous packages and your system likely has them already`?  I don't want people to see something that requires `sudo` and stopping right there.